docs/api.md

```diff
@@ -51,13 +51,13 @@ An array of functions to be called after Electron has been extracted to a tempor
 
 When `true`, sets both [`arch`](#arch) and [`platform`](#platform) to `all`.
 
-##### `app-copyright`
+##### `appCopyright`
```

Comment:
For the docs, what I typically do when I'm renaming an option is add an entry for the old name with the type and a deprecated message (for example: [`version-string`](https://github.com/electron-userland/electron-packager/blob/v8.5.1/docs/api.md#version-string))